zuul.d/nodeset.yaml

```
@@ -170,3 +170,151 @@
       - name: ocps
         nodes:
           - crc
+
+- nodeset:
+    name: centos-9-medium-crc-extracted-2-36-0-3xl
+    nodes:
+      - name: controller
+        label: cloud-centos-9-stream-tripleo-medium
+      - name: crc
+        label: coreos-crc-extracted-2-36-0-3xl
+    groups:
+      - name: computes
+        nodes: []
+      - name: ocps
+        nodes:
+          - crc
+
+- nodeset:
+    name: centos-9-crc-2-36-0-xxl
+    nodes:
+      - name: controller
+        label: centos-9-stream-crc-2-36-0-xxl
+
+- nodeset:
+    name: centos-9-rhel-9-2-crc-extracted-2.36-3xl
```

```suggestion
    name: centos-9-rhel-9-2-crc-extracted-2-36-0-3xl
```

This looks to be an error in the naming, I see it was also in the 2.30 nodeset as well ldenny~~ :P